index.js

```diff
@@ -1,13 +1,130 @@
-const homeMenu = document.getElementById('humMenu');
 const navCo = document.getElementById('nav_list_container');
 const closeBtn = document.getElementById('close-btn');
+const homeMenu = document.getElementById('humMenu');
 const body = document.getElementById('body');
 const homeBtn = document.getElementById('home-btn');
 const projectsBtn = document.getElementById('projects-btn');
 const aboutBtn = document.getElementById('about-btn');
 const contactBtn = document.getElementById('contact-btn');
 const headerDesign = document.getElementById('shape2');
 const shape1 = document.getElementById('shape1');
+const popUpWindow = document.getElementById('popup-window');
+const popupCloseBtn = document.getElementById('popup-close-btn');
+const card = document.querySelectorAll('.card');
+
+const projectData = [
+  {
+    seeProjectBtn: document.getElementById('project-btn1'),
+    name: 'Project name goes here',
+    description: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nesciunt explicabo, quidem dolorum totam consequuntur vitae consectetur, odio asperiores voluptas magni tenetur vel. Accusamus ut, minima quas aliquid consectetur cumque molestiae at modi fugit dolores ipsum, perferendis a quam saepe et doloremque quae reiciendis id, assumenda rerum. Fuga voluptatum earum aliquam?',
+    image: document.createElement('IMG'),
+    image_src: document.getElementById('project_img1').src,
+    list1: 'HTML/CSS',
+    list2: 'Ruby on Rail',
+    list3: 'Javascript',
+    p: 'see this project',
+  },
+  {
+    seeProjectBtn: document.getElementById('project-btn2'),
+    name: 'Project name goes here',
+    description: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nesciunt explicabo, quidem dolorum totam consequuntur vitae consectetur, odio asperiores voluptas magni tenetur vel. Accusamus ut, minima quas aliquid consectetur cumque molestiae at modi fugit dolores ipsum, perferendis a quam saepe et doloremque quae reiciendis id, assumenda rerum. Fuga voluptatum earum aliquam?',
+    image: document.createElement('IMG'),
+    image_src: document.getElementById('project_img2').src,
+    list1: 'HTML/CSS',
+    list2: 'Ruby on Rail',
+    list3: 'Javascript',
+    p: 'see this project',
+  },
+  {
+    seeProjectBtn: document.getElementById('project-btn3'),
```

Comment:
* Kindly adjust the popup on mobile and adjust the buttons to align in the center 
* also kindly stick to the color provided on the figma design for your text 
* Kindly reduce the popup size on desktop as users will have to scroll down to see the whole popup
![image](https://user-images.githubusercontent.com/52497246/224117803-136ef522-165a-4a36-8b13-30526ef2a64c.png)
![image](https://user-images.githubusercontent.com/52497246/224117922-be9a7598-21a7-40fc-8a94-d75cd59964f8.png)

Reply:
- [ ] Amazing :bomb: :clap: You did it!! Now, all that's left is to align the buttons at the center of the page on a mobile screen, just as the previous reviewer suggested :wrench: 
- This will improve your user interface, as well as match the Figma wireframe.

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/224301820-d4a84b3e-2d05-4474-bd7c-9fb845a3cb9f.png)|![image](https://user-images.githubusercontent.com/99715640/224301879-4a9ef822-476c-41e3-bb4a-e20b90ab960b.png)|